### PR TITLE
Migrate github actions to .NET 8 

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: .github/workflows/Lib edFi.admin.dataaccess pullrequest.yml [hunk 1]
```diff
@@ -19,11 +19,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-    - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
-      with:
-        dotnet-version: 6.0.x
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```
Comment (simpat-jesus) [inline .github/workflows/Lib edFi.admin.dataaccess pullrequest.yml:22]: The step was partially removed.

File: .github/workflows/Lib edFi.common pullrequest.yml [hunk 2]
```diff
@@ -18,11 +18,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-    - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
-      with:
-        dotnet-version: 6.0.x
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```
Comment (simpat-jesus) [inline .github/workflows/Lib edFi.common pullrequest.yml:21]: The step was partially removed.

File: .github/workflows/Lib edFi.loadtools pullrequest.yml [hunk 3]
```diff
@@ -19,11 +19,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-    - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
-      with:
-        dotnet-version: 6.0.x
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```
Comment (simpat-jesus) [inline .github/workflows/Lib edFi.loadtools pullrequest.yml:22]: The step was partially removed.

File: .github/workflows/Lib edFi.ods.api pullrequest .yml [hunk 4]
```diff
@@ -18,11 +18,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-    - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
-      with:
-        dotnet-version: 6.0.x
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```
Comment (simpat-jesus) [inline .github/workflows/Lib edFi.ods.api pullrequest .yml:21]: The step was partially removed.

File: .github/workflows/Lib edFi.ods.common pullrequest.yml [hunk 5]
```diff
@@ -18,11 +18,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-    - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
-      with:
-        dotnet-version: 6.0.x
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```
Comment (simpat-jesus) [inline .github/workflows/Lib edFi.ods.common pullrequest.yml:21]: The step was partially removed.

File: .github/workflows/Lib edFi.security.dataaccess pullrequest.yml [hunk 6]
```diff
@@ -19,11 +19,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-    - name: Setup .NET
-      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
-      with:
-        dotnet-version: 6.0.x
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
```
Comment (simpat-jesus) [inline .github/workflows/Lib edFi.security.dataaccess pullrequest.yml:22]: The step was partially removed.

**Review by simpat-jesus — CHANGES_REQUESTED** [review]

File: .github/workflows/Lib edFi.common pullrequest.yml (hunk 2 above)
Comment (semalaiappan) [reply .github/workflows/Lib edFi.common pullrequest.yml:21]: https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/actions/runs/7466652036/job/20318481037


(https://ed-fi-alliance.slack.com/archives/D04G1UMG3BM/p1704993915723789)
Reason I have this     - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4

Becuase  I need to checkout ODS repo for Restore NuGet packages step

**Review by simpat-jesus — APPROVED** [review]

**Pull request merged**